From 775b2a42d74bf6230e6395712a834b44318354e6 Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Thu, 6 Jun 2013 16:22:59 -0700
Subject: correctly check for ro.boot.dmi_detect

We want to skip this if it is set to 0, not the opposite.

Change-Id: Ic484550d8428472d472e038f8b297c509dcff94e
Signed-off-by: Andrew Boie <andrew.p.boie@intel.com>
---
 init/property_service.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/init/property_service.c b/init/property_service.c
index c2efae9..aa306e7 100755
--- a/init/property_service.c
+++ b/init/property_service.c
@@ -587,7 +587,7 @@ static void dmi_detect_props(void)
 
     /* Skip if disabled */
     var = property_get("ro.boot.dmi_detect");
-    if(var && strcmp(var, "0"))
+    if(var && !strcmp(var, "0"))
         return;
 
     /* Fail silently (no error) on devices without DMI */
-- 
1.7.4.1

